realm/realm-jni/build.gradle

```diff
@@ -81,6 +89,34 @@ if (project.clang instanceof String) {
 if (project.forceDownloadCore instanceof String) {
     ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
 }
+if (project.stripSymbols instanceof String) {
```

Comment:
I think all of these `instanceof` should be removed in favor of:

```groovy
ext.stripSymbols = project.stripSymbols as Boolean
```

What do you think @zaki50 ?